metadata_scripts/GCEMetadataScripts/main.go

```diff
@@ -258,7 +269,16 @@ func getMetadata() (map[string]string, error) {
 }
 
 func getScripts(mdsm map[metadataScriptType]string) ([]metadataScript, error) {
-	md, err := getMetadata()
+	md, err := getMetadata("/instance/attributes")
+	if err != nil {
+		return nil, err
+	}
+	msdd := parseMetadata(mdsm, md)
```

Comment:
If `serial-port-enable=True` is the entirety of instance level metadata, will a startup script run in project level metadata? I don't see where `parseMetadata` checks for `startup-script` (and other valid key names).

Reply:
parseMetadata is given map[metadataScriptType]string which it uses to parse for startup script metadata keys and returns the order to run them in. So no, no other metadata key should get parsed, I made the test check for this as well